journal/modernized/02-chapter-2.adoc

```diff
@@ -407,7 +407,9 @@ Thomas Greer to John Conran.
 [.signed-section-context-open]
 Stockton, Fourth Month 17th, 1780.
 
-Dear John, As I have passed along from place to place in this country,
+Dear John, 
```

Comment:
This is good, but we should also add the `[.salutation]` special instruction above it.  This is sort of advanced, so I'm not at all surprised you didn't do it on your first task. 😊 Here's what it would look like:

```ASCIIDOC
[.salutation]
Dear John,

As I have passed along...
```

Here's the section in the styleguide: https://flp-styleguide.netlify.com/#letters

Reply:
I added a line here so that there was a space between [.salutation] and "Stockton, Fourth Month..." above it. Is that okay? Wasn't sure if I should've just used the space already given.

Reply:
Kristi, yup, you did it right here, there should always be a blank line above the `[.<something>]` type lines, so this is perfect. 👍

And, nice job leaving a comment, you did it perfect!